Cargo.toml

```diff
@@ -61,11 +61,16 @@ exclude = [ "neard" ]
 
 [patch.crates-io]
 
+# Note that "bench" profile inherits from "release" profile and
+# "test" profile inherits from "dev" profile.
+# https://doc.rust-lang.org/cargo/reference/profiles.html#test
+
 [profile.release]
 overflow-checks = true
+panic = 'abort'
```

Comment:
I don't think we should do it for release. Nearcore should tolerate crashes in some cases. For example if one jsonrpc thread panics or if one peer actor thread panics it should not cause neard to shut down completely.

Reply:
If panic occurs, then it means that there is a bug in the implementation and it is dangerous to run incorrect code. A failed jsonrpc should return an error, not panic. If an actor panics silently, you end up with a crippled process which is very hard to debug.